tomcat/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -0,0 +1,60 @@
+package com.techcourse.controller;
+
+import com.techcourse.db.InMemoryUserRepository;
+import com.techcourse.model.User;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import org.apache.controller.AbstractController;
+import org.apache.coyote.http11.request.Http11Request;
+import org.apache.coyote.http11.response.Http11Response;
+import org.apache.coyote.session.Session;
+import org.apache.util.QueryStringParser;
+
+public class LoginController extends AbstractController {
+
+    public LoginController() {
+        super.endPointPattern = Pattern.compile("^/login");
+    }
+
+    @Override
+    protected void doPost(Http11Request request, Http11Response response) throws Exception {
+        String body = request.getBody();
+        Map<String, List<String>> queryStrings = QueryStringParser.parseQueryString(body);
+        String account = queryStrings.get("account").getFirst();
```

Comment:
```suggestion
        String account = queryStrings.get("account");
```

account가 없으면 에러 터지는데 어떻게 getFirst()를 바로 쓰시죠?

Reply:
QueryString 예외 처리를 추가했습니다.